client/obclient.go

```diff
@@ -50,6 +50,7 @@ type obClient struct {
 	password     string
 	database     string
 	sysUA        *route.ObUserAuth
+	entityType   protocol.ObTableEntityType
```

Comment:
how about default value？ Maybe you should set default value = KV

Reply:
set  default value = KV when init obClient 
![image](https://github.com/oceanbase/obkv-table-client-go/assets/34029771/ac3aee7c-0fd6-4108-97f8-7a57a22be1cd)